containers/signing-service/Dockerfile.j2

```diff
@@ -3,19 +3,19 @@ FROM debian:{{ debian_version }}
 COPY create-user.sh                     /tmp/
 RUN  /tmp/create-user.sh
 
-COPY signing-service/RELEASE-VERSION    /usr/lib/signing_service/RELEASE-VERSION
-COPY signing-service/requirements.lock  /usr/lib/signing_service/signing-service/
-COPY testing.pref                       /etc/apt/preferences.d/
-COPY build-dependencies.txt             /tmp/
-COPY install-dependencies.sh            /tmp/
-COPY install-signing-service.sh         /tmp/
-COPY clean-container.sh                 /tmp/
-COPY middleman_protocol/                /usr/lib/signing_service/middleman_protocol/
-COPY signing-service/RELEASE-VERSION    /usr/lib/signing_service/middleman_protocol/RELEASE-VERSION
+COPY signing-service/cacheable-requirements.lock  /usr/lib/signing_service/signing-service/
+COPY testing.pref                                 /etc/apt/preferences.d/
+COPY build-dependencies.txt                       /tmp/
+COPY install-dependencies.sh                      /tmp/
```

Comment:
Python requirements should be after system dependencies. Otherwise you'll be reinstalling system packages every time the Python dependencies change - which is completely unnecessary.